dashboard/modules/metrics/grafana_dashboard_factory.py

```diff
@@ -126,6 +126,10 @@ class Panel:
                 expr='sum(ray_resources{{Name="CPU",{global_filters}}})',
                 legend="MAX",
             ),
+            Target(
+                expr='(sum(ray_resources{{Name="CPU",{global_filters}}}) or vector(0)) + (sum(autoscaler_pending_resources{{resource="CPU",{global_filters}}}) or vector(0))',
```

Comment:
or vector(0) is added because otherwise when we scale up from 0 -> N CPU/GPU, the timeseries doesn't come up because it is considered as an error query (since the metrics don't exist)